oauth2_provider/__init__.py

```diff
@@ -1,3 +1 @@
 __version__ = "1.5.0"
-
-default_app_config = "oauth2_provider.apps.DOTConfig"
```

Comment:
something like https://github.com/celery/django-celery-beat/blob/master/django_celery_beat/__init__.py#L34 would be better, https://github.com/celery/django-celery-beat/blob/master/django_celery_beat/__init__.py#L9

Reply:
https://github.com/jazzband/django-oauth-toolkit/pull/1035/commits/cd4492344a5c12dc7aea1431ace703ce709e8dd4